lib/Runtime/Library/BoundFunction.cpp

```diff
@@ -171,9 +172,20 @@ namespace Js
             }
 
             // Copy the bound args
-            for (uint i=0; i<boundFunction->count; i++)
+            if (!isCrossSiteObject)
             {
-                newValues[index++] = boundFunction->boundArgs[i];
+                for (uint i = 0; i < boundFunction->count; i++)
```

Comment:
Though its not a common usage in current code but why not replace if else with
_newValues[index++] = !boundFunction->IsCrossSiteObject() ? boundFunction->boundArgs[i] : CrossSite::MarshalVar(scriptContext, boundFunction->boundArgs[i]);_
keeps assignment in one place